_tools/check_docs.ts

```diff
@@ -202,6 +202,10 @@ function assertHasParamDefinition(
       paramDoc.kind === "assign" && paramDoc.left.kind === "identifier"
     ) {
       return paramDoc.left.name === param.name;
+    } else if (
+      paramDoc.kind === "assign" && paramDoc.left.kind === "object"
+    ) {
+      return true;
```

Comment:
This change has undesirable effect in doc rendering.

By using destructuring in parameter, there's no longer `options` param in arguments, and `@param options` directive becomes unclear about what it documents.

Reply:
I see. Why is it though that `deno task lint:docs` allows https://github.com/denoland/std/blob/cc9a7c006d3587c4eee04f4924dfe8c2cc928962/cli/unstable_prompt_select.ts#L41 but will throw here https://github.com/denoland/std/blob/cc9a7c006d3587c4eee04f4924dfe8c2cc928962/cli/unstable_prompt_multiple_select.ts#L44